spec.md

```diff
@@ -2791,10 +2791,19 @@ They are interpreted according to Starlark's [indexing conventions](#indexing).
 <a id='string·endswith'></a>
 ### string·endswith
 
-`S.endswith(suffix)` reports whether the string S has the specified suffix.
+`S.endswith(suffix[, start[, end]])` reports whether the string 
+`S[start:end]` has the specified suffix.
 
 ```python
 "filename.sky".endswith(".sky")         # True
+"filename.sky".endswith(".sky", 9, 12)  # False
```

Comment:
Add an exemple with start/end where it returns True. This can help understand how it works.